icinga/commands-jitsi.conf

```diff
@@ -35,6 +35,4 @@ object CheckCommand "jitsi" {
         description = "Blacklisted metrics, which should be ignored in performance data"
     }
   }
-
-  vars.jitsi_host = "127.0.0.1"
```

Comment:
Please use ``vars.jitsi_host = "$host.address$"`` instead of hardcoding the value above. This allows the user to override the value if the address ist not the same as the hosts address (e.g private subnet or sth else)